browser/css/jsdialogs.css

```diff
@@ -1510,10 +1510,22 @@ input[type='number']:hover::-webkit-outer-spin-button {
 	margin: auto;
 }
 
+[data-theme='dark'] .ui-image {
+	filter: invert(.95)
+}
+
```

Comment:
This can enter but I'm afraid of being too generic and ending up inverting images that shouldn't be inverted (either because will look odd, or because they are already in dark mode). Better to add `#PageNumberDialog` before `.ui-image`